### ISSUE TITLE
Make hot walls glow

### ISSUE THREAD

**Issue by Sebeb** [dhubber/mazed-up#24]
It would be great if we could add a bloom effect only to the walls when the hot walls pill has been taken. There must be a good, basic shader which will do the trick somewhere out there

**Comment by dhubber** [comment]
Yes, the new hot walls isn't as clear as the old one (which was perhaps too strong) so we need something more for this.  I didn't even realise I'd had the hot walls pill until I started hearing the 'sizzling' noise!
